src/main/java/com/example/school/SchoolApplication.java

```diff
@@ -1,18 +1,52 @@
 package com.example.school;
 
+import com.example.school.model.Group;
 import com.example.school.model.Student;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
 import java.time.OffsetDateTime;
+import java.util.ArrayList;
+
+import java.util.List;
+import java.util.UUID;
 
 @SpringBootApplication
 public class SchoolApplication {
 
     public static void main(String[] args) {
         SpringApplication.run(SchoolApplication.class, args);
-        Student student1 = new Student("Igor", "Petrov", OffsetDateTime.parse("2022-09-01T09:00:00+01:00"));
-        Student student2 = new Student("Oleg", "Ivanov", OffsetDateTime.parse("2022-09-01T09:00:00+01:00"));
-        Student student3 = new Student("Maxim", "Svetlov", OffsetDateTime.parse("2022-09-01T09:00:00+01:00"));
+
+        var group1A = new Group(UUID.randomUUID(), "1A", new ArrayList<>());
```

Comment:
It would be better use `@Builder`(from lombok) every where instead of contractor.